modules/tile-converter/src/i3s-converter/helpers/geometry-converter.ts

```diff
@@ -1,4 +1,7 @@
 import type {Image, MeshPrimitive} from 'modules/gltf/src/lib/types/gltf-postprocessed-schema';
+import type {B3DMContent, FeatureTableJson} from '@loaders.gl/3d-tiles';
+import type {GLTF_EXT_feature_metadata} from '@loaders.gl/gltf';
+import TileHeader from '@loaders.gl/tiles/src/tileset/tile-3d';
```

Comment:
Let's not do path imports. You need to export the symbol, if internal use underscore when exporting.

Reply:
Done